test/utils/helperFn.ts

```diff
@@ -76,7 +78,7 @@ export function getColumnHeadersTextContent() {
 
 export function getCell(rowIndex: number, colIndex: number): HTMLElement {
   const cell = document.querySelector(
-    `[role="cell"][data-rowindex="${rowIndex}"][aria-colindex="${colIndex}"]`,
+    `[role="cell"][data-rowindex="${rowIndex}"][data-colindex="${colIndex}"]`,
```

Comment:
ARIA > implementation details of the implementation

```suggestion
    `[role="cell"][data-rowindex="${rowIndex}"][aria-colindex="${colIndex + 1}"]`,
```

Reply:
I had went with @dtassone 's suggestion considering zero-based with `data`: https://github.com/mui-org/material-ui-x/pull/1669#discussion_r633337082. 
> to not leak implementation details on the public API

What are we leaking? `data` attribute?

Reply:
For the confusing aspect, I had assumed that this aspect is hidden to the maintainers writing tests, you don't need to know about it. We could add a comment to explain the rational.

For testing the a11y, this is the beauty of using aria, there are fewer tests to write.

> What are we leaking? data attribute?

Yes, this is the other aspect that can be challenged. We are now providing the developers two different and non consistent ways to test their data grid. They can either use aria or data, both that use two different references. They have to decide which one to choose.

Reply:
> We are now providing the developers two different and non consistent ways to test their data grid. They can either use aria or data, both that use two different references. They have to decide which one to choose.

Hmm, okay. I see. They will have to know `aria` is 1-based while `data` is 0-based if they retrieve for tests.